src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1213,7 +1214,19 @@ static bool addTriggeredSubscriptions_withCache
       LM_T(LmtMongo, ("adding subscription: '%s'", sub.toString().c_str()));
 
       long long throttling       = sub.hasField(CSUB_THROTTLING) ? getField(sub, CSUB_THROTTLING).numberLong() : -1;
-      long long lastNotification = sub.hasField(CSUB_LASTNOTIFICATION) ? getIntField(sub, CSUB_LASTNOTIFICATION) : -1;
+      long long lastNotification = -1;
+
+      if (sub.hasField(CSUB_LASTNOTIFICATION))
```

Comment:
Pontentially, we could have the same problem for NGSI9 subscription (look for CASUB_LASTNOTIFICATION field). I wonder it is is worth the effort also to implement a similar fix for them (probably simpler, as casubs doesn't use yet a cache). The point to take into account is that NGSI9 subscription are not very used nowadays.